config/detekt/detekt.yml

```diff
@@ -740,6 +740,7 @@ style:
   UnusedPrivateMember:
     active: true
     allowedNames: ''
+    ignoreAnnotated: ['Preview']
```

Comment:
Following the [official suggestions](https://detekt.dev/docs/introduction/compose/#unusedprivatemember-for-compose) from detekt's documentation around Compose, ignore `@Preview` annotated functions for the `UnusedPrivateMember` rule